### DISCUSSION TITLE
Devices, how to use them?

### DISCUSSION THREAD

**Discussion by pdo59** [Part-DB/Part-DB-server#188] [Q&A] [answered]
I would like to know how to use the devices: once I have created a device what can I do with it?

**Comment by jbtronics** [comment, marked as answer]
Devices are not implemented yet, so they do not work.
The whole device stuff you can currently see, is for backwards compatibility with the old Part-DB version (so that you at least can list your old devices). Creation of new devices currently is not very useful.

Actually it might make sense, so disable the device sections by setting the device permissions of a user to disallow.

EDIT: Devices (now called projects) are now implemented and work.

**Reply by pdo59** [reply]
Ok, thank you.

**Reply by virtadpt** [reply]
Now things make more sense.  Thank you.
